src/main/java/com/selfxdsd/selfweb/api/WalletsApi.java

```diff
@@ -165,9 +165,12 @@ public WalletsApi(final User user, final Self self) {
         @PathVariable final String owner,
         @PathVariable final String name,
         @PathVariable final String type,
-        @RequestBody @Positive final float limit) {
+        @RequestBody  final float limit) {
         final ResponseEntity<String> response;
-        if (type.equals(Wallet.Type.FAKE)) {
+        if(limit < 0){
```

Comment:
@amihaiemil I've ditched `@Posivitive` validation since it req to add `Errors` as parameter too, thus breaking the tests. So I just manually validated on-site + added unit test for.

Reply:
@amihaiemil Should we allow users to put limit 0?

Reply:
@criske I'm not sure I understand:

> I've ditched @Positive validation since it req to add Errors as parameter too, thus

What is ``Errors`` and where is it required as parameter? :D

Reply:
@criske it's ok if we don't allow 0 for now.

Reply:
> What is `Errors` and where is it required as parameter? 

@amihaiemil I've read somewhere on stack that [Errors](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/validation/Errors.html) is needed in order to validation to kick in. But I don't think that's the case,  didn't tested though.
But now I've checked the docs and `@Validated` is requiered at class level and it works fine. The problem is that is throwing `ConstraintViolationException` and is not catched by `handleValidationExceptions` handler from `BaseApiController`. 
Should I go back to `@Positive`? :D

Reply:
@criske yes, please go back to it. I've never heard of any ``Errors`` thing :D 
And leave a puzzle for handling ConstraintViolation properly.